package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@supabase/pg-api",
-  "version": "0.0.9",
+  "version": "0.0.0",
```

Comment:
Is this correct @soedirgo ? Should it be back at 0?

Reply:
Hmm, maybe `0.0.0-automated` is more explicit? The `version` field is unused since `semantic-release` infers the current version from the latest tag.